sdk/inc/azure/core/az_mqtt5.h

```diff
@@ -208,6 +208,11 @@ typedef struct
    * @brief The publish request ID.
    */
   int32_t id;
+
+  /**
+   * @brief Publish acknowledgement (PUBACK) reason code. Indicates success or reason for failure.
+   */
+  int32_t puback_reason;
```

Comment:
```suggestion
  int32_t reason_code;
```

Reply:
"reason code" relates more to how it's called in the spec.
https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901124

Reply:
@RLeclair I think these changes make more sense to address in your PR (see the other comments from Ewerton as well)

Reply:
Sync'd with Raul and he'll address these naming comments in his mqtt related PR instead of me making these changes in the RPC client PR.